### PR TITLE
Multimodal Neurons in Artificial Neural  Networks

### PR DESCRIPTION
**Pull request by wilkistermumbi**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/multimodal-neurons-in-artificial-neural-networks/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+They found neurons that respond to words, facial expressions, and any content associated with an emotional or mental state. These emotional states include sadness, surprise, shock, crying, happiness, and sleepiness. This example is shown below:
+
+![Emotion Neuron](/engineering-education\multimodal-neurons-in-artificial-neural-networks/emotion-neuron.PNG)
+
+*[Image Source: Distill](https://distill.pub/2021/multimodal-neurons/)*
+
+### Typographic Attacks
+
+1. The model responds heavily to rendered text. This can be seen from the adversarial attacks where, i.e., take an apple and attach a sticker labeled iPod on it, it labels the picture as an iPod instead of an apple. This example shows that the text might still be too dominant in this model. They also show that randomly rendering texts on images confuse the model. For example, rendering texts of "pizza" on top of a dog image confuses the classifier by making it classify the picture as "pizza" instead of a "dog."
```
Comment (collins-a) [inline articles/multimodal-neurons-in-artificial-neural-networks/index.md:72]: This section feels a bit disjointed from the rest of the article. For instance, a reader may wonder what model is being referred to by "the model". As such, introduce the section in a few sentences to maintain the flow of the article.

**Review by collins-a — COMMENTED** [review]
@wilkistermumbi good job introducing multimodal neurons in ANNs. Just a minor change and we'll be ready for a final review. Plus, I suggest working on giving more of your voice to your future articles. Cheers

**Comment by wilkistermumbi** [comment]
@collins-a I've noted your review. I'm done with the requested changes

**Review by collins-a — APPROVED** [review]
@wilkistermumbi I'm satisfied with the changes. The article is ready for the final review.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**